nflow-rest-api-common/src/main/java/io/nflow/rest/v1/ResourceBase.java

```diff
@@ -183,15 +183,22 @@ public ListWorkflowInstanceResponse fetchWorkflowInstance(final long id, final S
     return listWorkflowConverter.convert(instance, includes);
   }
 
+  protected int resolveExceptionHttpStatus(Throwable t) {
+    if (t instanceof IllegalArgumentException) {
+      return 400;
```

Comment:
Not added in this PR, but prefer using some existing constants for HTTP status codes. Same for 404 and 500.

Reply:
Sure - fixing.